website/snippets/_new-sl-setup.md

```diff
@@ -1,31 +1,89 @@
-    * Team plan &mdash; Owner with a Developer license.
-    * If you are using a free trial dbt Cloud account, you are on a trial of the Team plan as an Owner, so you're good to go.
-- You must have a successful run in your new environment.
+### 1. Select environment
 
-Now that we've created and successfully run a job in your environment, you're ready to configure the semantic layer.
+Select the environment where you want to enable the Semantic Layer:
 
-1. Navigate to **Account Settings** in the navigation menu.
-2. Use the sidebar to select your project settings. Select the specific project you want to enable the Semantic Layer for.
-3. In the **Project Details** page, navigate to the **Semantic Layer** section, and select **Configure Semantic Layer**.
+1. Navigate to **Account settings** in the navigation menu.
+2. On the **Settings** left sidebar, select the specific project you want to enable the Semantic Layer for.
+3. In the **Project details** page, navigate to the **Semantic Layer** section. Select **Configure Semantic Layer**.
 
-<Lightbox src="/img/docs/dbt-cloud/semantic-layer/new-sl-configure.jpg" width="60%" title="Semantic Layer section in the Project Details page"/>
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/new-sl-configure.jpg" width="60%" title="Semantic Layer section in the 'Project Details' page"/>
 
-4. In the **Set Up Semantic Layer Configuration** page, enter the credentials you want the Semantic Layer to use specific to your data platform.
+4. In the **Set Up Semantic Layer Configuration** page, select the deployment environment you want for the Semantic Layer and click **Save**. This provides administrators with the flexibility to choose the environment where the Semantic Layer will be enabled.
 
-     - Use credentials with minimal privileges. This is because the Semantic Layer requires read access to the schema(s) containing the dbt models used in your semantic models for downstream applications
-     - Note, [Environment variables](/docs/build/environment-variables) such as `{{env_var('DBT_WAREHOUSE')}`, doesn't supported the dbt Semantic Layer yet. You must use the actual credentials.
+### 2. Add a credential
+The dbt Semantic Layer uses [service tokens](/docs/dbt-cloud-apis/service-tokens) for authentication, mapped to an underlying data platform credential. This credential has physical access to the raw data, meaning it controls the data accessed by the Semantic Layer.
 
-<Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl-configure-sl.jpg" width="45%" title="Enter the credentials you want the Semantic Layer to use specific to your data platform, and select the deployment environment."/>
+dbt Cloud Enterprise plans can add multiple credentials and map those to service tokens. Refer to [Add more credentials](#4-add-more-credentials) for more information.
```

Comment:
hey @rpourzand, to address your comment below on enterprise creating a token in the same flow (not having to go back to details page), i meant to use the below sentence to tell enterprise users to go to the 'add more credentials section' since those steps/new ui apply to them. 

how about i:
- turn the sentence into a callout
- add the callout before the 'add a credential' section.
- make it more explicit, like: dbt Cloud Enterprise plans can add multiple credentials and have a different set up. Skip to [Add more credentials](#4-add-more-credentials) for more configuration details.

so 

4. In the Set Up Semantic Layer Configuration page, select the deployment environment you want for the Semantic Layer and click Save. This provides administrators with the flexibility to choose the environment where the Semantic Layer will be enabled.

:::tip dbt Cloud Enterprise can skip to [Add more credentials](#4-add-more-credentials)
dbt Cloud Enterprise plans can add multiple credentials and have a different set up. Skip to [Add more credentials](#4-add-more-credentials) for more configuration details.
:::

## 2. Add a credential..

Reply:
I made these changes in [this commit](https://github.com/dbt-labs/docs.getdbt.com/pull/5608/commits/54e4d8836f7617307eccc225377b32adaa210d2f)